_posts/2016-09-30-sr2017-kickstart-dates-announced.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: news
+title: 2017 Kickstart locations announced
+---
+
+We are happy to announce the locations for our 2017 Kickstart on the 15th of
+ October 2016.
+
+Kickstart is the event which kicks off a competition year, where the game will
+be announced, and robotics kits will be handed out to the teams.
+
+The venues for Student Robotics 2017 kickstart are:
+
+- Southampton University
+- Bristol University
+- The Kings Centre, Oxford
+
+Please visit our [Kickstart page][kickstart-page] for more information. If you
+want to take part, but haven't already signed up to compete, you can join our
+waiting list by [starting a team][compete].
+
+[kickstart-page]: {{ '/2017/kickstart' | prepend: site.baseurl }}
+[compete]: {{ '/compete' | prepend: site.baseurl }}
```

Comment:
Surely these could just be of the form `{{ site.baseurl }}/compete`?

Not sure which is preferred, though the format I describe seems simpler to me.